### ISSUE TITLE
Loss doesn't decrease

### ISSUE THREAD

**Issue by Iceland-Leo** [Vision-CAIR/MiniGPT-4#67]
@TsuTikgiau I just use your stage2 pretrained weight to finetune on my own dataset, but the loss is always 10.3750. Does it meet expectation?
Train: data epoch: [0]  [3950/5000]  eta: 0:08:37  lr: 0.000029  loss: 10.3750  time: 0.4920  data: 0.0000  max mem: 43374
Train: data epoch: [0]  [4000/5000]  eta: 0:08:12  lr: 0.000029  loss: 10.3750  time: 0.4583  data: 0.0000  max mem: 43374
Train: data epoch: [0]  [4050/5000]  eta: 0:07:47  lr: 0.000029  loss: 10.3750  time: 0.4734  data: 0.0000  max mem: 43374
Train: data epoch: [0]  [4100/5000]  eta: 0:07:22  lr: 0.000029  loss: 10.3750  time: 0.4808  data: 0.0000  max mem: 43374
Train: data epoch: [0]  [4150/5000]  eta: 0:06:58  lr: 0.000029  loss: 10.3750  time: 0.4960  data: 0.0000  max mem: 43374
Train: data epoch: [0]  [4200/5000]  eta: 0:06:33  lr: 0.000029  loss: 10.3750  time: 0.4918  data: 0.0000  max mem: 43374
Train: data epoch: [0]  [4250/5000]  eta: 0:06:09  lr: 0.000029  loss: 10.3750  time: 0.4753  data: 0.0000  max mem: 43374
Train: data epoch: [0]  [4300/5000]  eta: 0:05:44  lr: 0.000029  loss: 10.3750  time: 0.4894  data: 0.0000  max mem: 43374
Train: data epoch: [0]  [4350/5000]  eta: 0:05:19  lr: 0.000029  loss: 10.3750  time: 0.4882  data: 0.0000  max mem: 43374
Train: data epoch: [0]  [4400/5000]  eta: 0:04:55  lr: 0.000029  loss: 10.3750  time: 0.4856  data: 0.0000  max mem: 43374
Train: data epoch: [0]  [4450/5000]  eta: 0:04:30  lr: 0.000028  loss: 10.3750  time: 0.4785  data: 0.0000  max mem: 43374
Train: data epoch: [0]  [4500/5000]  eta: 0:04:05  lr: 0.000028  loss: 10.3750  time: 0.4981  data: 0.0000  max mem: 43374
Train: data epoch: [0]  [4550/5000]  eta: 0:03:41  lr: 0.000028  loss: 10.3750  time: 0.5036  data: 0.0000  max mem: 43374
Train: data epoch: [0]  [4600/5000]  eta: 0:03:16  lr: 0.000028  loss: 10.3750  time: 0.4828  data: 0.0000  max mem: 43374
Train: data epoch: [0]  [4650/5000]  eta: 0:02:52  lr: 0.000028  loss: 10.3750  time: 0.4872  data: 0.0000  max mem: 43374
Train: data epoch: [0]  [4700/5000]  eta: 0:02:27  lr: 0.000028  loss: 10.3750  time: 0.4992  data: 0.0000  max mem: 43374
Train: data epoch: [0]  [4750/5000]  eta: 0:02:02  lr: 0.000028  loss: 10.3750  time: 0.4900  data: 0.0000  max mem: 43374
Train: data epoch: [0]  [4800/5000]  eta: 0:01:38  lr: 0.000028  loss: 10.3750  time: 0.4916  data: 0.0000  max mem: 43374
Train: data epoch: [0]  [4850/5000]  eta: 0:01:13  lr: 0.000028  loss: 10.3750  time: 0.4964  data: 0.0000  max mem: 43374

**Comment by HellwayXue** [comment]
Hi, how did you solve that? I met the same issue.

**Comment by Rh-Dang** [comment]
I meet the same problem. I'm trying to read its loss computing code

**Comment by 1359347500cwc** [comment]
I also meet the same problem. I'm trying to read its loss computing code
